src/Integration/Integration.csproj

```diff
@@ -100,21 +88,6 @@
          EmbedInterAssemblies isn't set for PackagReferences. -->
     <PackageReference Include="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.27" />
   </ItemGroup>
-
-  <!-- The Alm.Auth assemblies aren't being included in the VSIX for some reason.
-       This workaround adds metadata to three artifacts so that the standard VSSDK targets will
-       include them.
-       NB the target is based on the one from Microsoft.VisualStudio.SDK.EmbedInteropTypes above. -->
-  <Target Name="HackIncludeALMAuthInVsix" AfterTargets="ResolveReferences" BeforeTargets="FindReferenceAssembliesForReferences">
-    <ItemGroup>
-      <ReferencePath Condition="'%(FileName)' == 'Microsoft.Alm.Authentication' or '%(FileName)' == 'Microsoft.Alm.Git' or '%(FileName)' == 'Microsoft.Vsts.Authentication'">
-        <CopyLocal>true</CopyLocal>
-      </ReferencePath>
-    </ItemGroup>
-
-    <Message Importance="high" Text="In HackIncludeAlmAuthInVsix" />
-    <Message Importance="high" Text="Reference paths: @(ReferencePath)" />
-  </Target>
```

Comment:
This hack was only necessary because the package reference in line #68 in this file specified `IncludeAssets="compile"` i.e. the assets should only be available at compile time, and not copied to the output folder. The hack was then effectively making sure they did get copied to the output folder, and also included in the VSIX.

However, just removing the `InlcudeAssets` attribute would have been enough.
I've checked that the three Alm.* assemblies are still included in the VSIX.